osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -123,16 +122,22 @@ protected override void LoadComplete()
             {
                 base.LoadComplete();
 
-                Combo.ValueChanged += _ => UpdateFlashlightSize(GetSize());
+                FlashlightSize = adjustCurrentSize();
```

Comment:
This is changing behaviour so should be double-checked.

Reply:
The behaviour is still the same, the only thing that changes was that this was being done on the constructors of every `ModFlashlight` implementation.

This defines the `FlashlightSize` so `updateFlashlightSize` works properly:
```cs
this.transformTo(nameof(FlashlightSize)...)
```

Reply:
Right, misread the variable names.